docker/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     volumes:
       - nodeguard_postgres_data:/var/lib/postgresql/data
     ports:
-      - 25432:5432
+      - 5432:5432
```

Comment:
i'd say let's keep nodeguard and our backend DBs separated

Reply:
Done 🏁